src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/CmdLineClient.kt

```diff
@@ -27,6 +33,33 @@ fun getRequestData(zipBytes: ByteArray, rootFileName: String, taskId: String): B
             .toByteString()
 }
 
+fun directoryExistingCheck(directory: File) {
```

Comment:
Can you add [`@Throws` annotation](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-throws/index.html)?

Reply:
Done